frontend/pages/queries/edit/components/DiscardDataOption/DiscardDataOption.tsx

```diff
@@ -84,6 +84,15 @@ const DiscardDataOption = ({
           </>
         </InfoBanner>
       )}
+      {selectedLoggingType === "" && (
```

Comment:
I think a cleaner fix here is just to update line 77 above to be `{selectedLoggingType !== "snapshot"} &&`... and to update the InfoBanner copy to be the more updated and general version you're adding here

Reply:
Or if @rachaelshaw wants to have both banner texts (specific to "differential" and more general "not set to snapshot"), have a ternary in the text

Reply:
Does `selectedLoggingType` come from the current dropdown selection? If so, couldn't we have the dropdown itself default to snapshot when it's empty string (rather than show that "Select" state from the screenshot)?

Reply:
@rachaelshaw someone correct me if I'm wrong, but the issue with showing snapshot in the dropdown when it is set to empty string on the backend, it's not running a snapshot

Reply:
I believe snapshot is the default (@lucasmrod @mostlikelee)?

Reply:
If not set, then the query is stored with empty `logging_type` in the database. Probably an oversight in #7765.

Reply:
@lucasmrod is the behavior the same as snapshot in that case? (If so, I think it's ok to default to snapshot in the UI, since saving the query should update it to _actually_ be snapshot.)

Reply:
We might need to cover the case where `logging_type` is set to empty `""` and change some logic to treat such value as if it were `"snapshot"`.